src/lib/rest/RestService.cpp

```diff
@@ -149,7 +149,7 @@ std::string payloadParse
 *
 * tenantCheck - 
 */
-static std::string tenantCheck(const std::string& tenant)
+extern std::string tenantCheck(const std::string& tenant)
```

Comment:
Maybe a note about reverting this citing #2781 should be added?

(In fact #2781 cite also another function, where this "rollback remark" could also make sense)